src/mixins/name-request-mixin.ts

```diff
@@ -60,16 +60,16 @@ export default class NameRequestMixin extends Vue {
    * @param nr the name request response payload
    * */
   isNrValid (nr: any): boolean {
-    let requestActionCDList = [NameRequestTypes.CHANGE_OF_NAME, NameRequestTypes.CONVERSION]
-    if (this.getResource.changeData) requestActionCDList = this.getResource.changeData.nameRequestTypes
```

Comment:
This was the real bug: `this.getResource.changeData` was not falsey, but `this.getResource.changeData.nameRequestTypes` was undefined; unfortunately the  old code tried to de-reference `undefined` on line 72.